phpBB/phpbb/console/command/searchindex/create.php

```diff
@@ -138,7 +138,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 				$this->state_helper->update_counter($status['post_counter']);
 
 				$progress->setProgress($status['post_counter']);
-				$progress->setMessage(round($status['rows_per_second'], 2) . ' rows/s');
+				$this->language->lang('CLI_ROWS_PER_SECOND', $progress->setMessage(round($status['rows_per_second'], 2)));
```

Comment:
```php
$progress->setMessage($this->language->lang('CLI_ROWS_PER_SECOND', round($status['rows_per_second'], 2)));
```

I haven't test, but this is how you should do, translate the content inside the setMessage